### PR TITLE
Clarify instructions and make identifiers legal

### PR DESCRIPTION
**Pull request by rrcobb**
Note: accidentally committed this against master, then reverted 🤕 

Adds improved instructions so that online students in particular have a chance at solving this without added clarification from instructors or AAQ.

Also updates the ids of the buttons so that they are valid HTML identifiers (`-`, `+`, and `<3` actually don't work! ht: @Joshua-Miles )

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,36 @@
...
+
+## Rewriting the functionality
+
+To begin, comment out the script tag linking to `index.min.js` in the html file.
+
+Write your code in `challenge.js` - it's already included in the HTML via a `<script>` tag, so when you add your code, it will run on the page.
+
+Add the following features one by one, refreshing the page to see your functionality working as you build.
+
+> Note: These deliverables are written in the form of _User Stories_. They describe the features from the perspective of a user when they visit the page.
```
Comment (ipc103) [inline README.md:22]: 👍

**Review by ipc103 — APPROVED** [review]
Thanks Rob - I think this should add enough clarity for our online students to be able to complete without additional human instruction. Thanks for getting these in so quickly.

**Review by alexgriff — APPROVED** [review]
Nice changes @rrcobb 
This was a one off thing made for one class (when i was in a particularly sadistic mood lol) and probably never even designed to be given without the context of someone walking students through the functionality in person.

I wonder if it's worth mentioning somewhere that this lab is really challenging, not getting everything done is ok, or something like that?

**Comment by rrcobb** [comment]
hmm, I think it might be okay without mentioning it explicitly. There aren't tests for the lab, I think the content is scaffolded pretty well, and there's the minified version to check in case anyone gets lost. With that many 'outs' I am hopeful that students will be fine.

**Pull request merged**